editor/components/post-author/check.js

```diff
@@ -27,17 +25,12 @@ export function PostAuthorCheck( { user, authors, children } ) {
 
 export default compose( [
 	withSelect( ( select ) => {
+		const post = select( 'core/editor' ).getCurrentPost();
 		return {
+			hasAssignAuthorAction: get( post, [ '_links', 'wp:action-assign-author' ], false ),
```

Comment:
I realised I forgot to comment on this in #6529. I think it'd be useful to create a selector for retrieving this data. Something like `select( 'core/editor' ).getPostLink( 'wp:action-assign-author' )`, or maybe `select( 'core/editor' ).getPostActionFlag( 'assign-author' )`. Neither of those names are good, but you get the idea.

Reply:
Cool, I've opened an issue for this #6655